pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -252,6 +256,15 @@ type KeyManagementServiceSpec struct {
 	TokenSecretName string `json:"tokenSecretName,omitempty"`
 }
 
+type ServerSideEncryptionType struct {
```

Comment:
```suggestion
type ServerSideEncryptionSpec struct {
```

The usage of `Spec` is more common across our API.